selfdrive/test/test_time_to_onroad.py

```diff
@@ -33,7 +33,7 @@ def test_time_to_onroad():
           sm.update(100)
 
           # sendcan is only sent once we're initialized
-          if sm.seen['controlsState'] and sendcan_frame is None:
+          if sm.seen['sendcan'] and sendcan_frame is None:
```

Comment:
it was hitting controlsInitializing very often with `controlsState`, plus the comment suggests that it should have been `sendcan`. But the fingerprinting uses sendcan, so it thinks it's initialized when it's not.